JavaScript/Question_03/question_03.js

```diff
@@ -0,0 +1,16 @@
+// Only change code below this line
+function countdown(myArray, n) {
+    if (n == 1) {
+        return [1];
+    } else if (n > 1) {
+        myArray = countdown(myArray, n - 1);
+        myArray.unshift(n);
+        return myArray;
+    }
+}
+
+// It should return [10, 9, 8, 7, 6, 5, 4, 3, 2, 1]
+console.log(countdown([],10));
+
+// It should return [5, 4, 3, 2, 1]
+console.log(countdown([],5));
```

Comment:
It is preferred to have a blank line at the end.
You have not added it.
Plus you have created a branch for each question. 
That was not what we mentioned in the [Contributing.md](https://github.com/DSC-SIST/D-Bug/blob/main/CONTRIBUTING.md#step-2--branch) file.
You will be pushing your solutions in the branch with your name.